lib/chutney/TorNet.py

```diff
@@ -763,6 +766,32 @@ def _genRouterKey(self):
                   .format(" ".join(cmdline), stdouterr))
             sys.exit(1)
         self._env['fingerprint'] = fingerprint
+       
+
+    def _setEd25519Id(self):
+        """Read the ed25519 identity key for this router, and set up the 'ed25519-id' entry in the Environ"""
+        datadir = self._env['dir']
+        key_directory = os.path.join(datadir, 'keys', "ed25519_master_id_public_key")
+        EXPECTED_ED25519_FILE_SIZE = 64
+        CURRENT_FILE_SIZE = os.stat(key_directory).st_size
+        if os.path.exists(key_directory) == False:
+            raise ValueError("File does not exist")
+        elif CURRENT_FILE_SIZE != EXPECTED_ED25519_FILE_SIZE:
+            raise ValueError("The current size of the file is {} bytes, which is not matching the expected value of {} bytes".format(CURRENT_FILE_SIZE,EXPECTED_ED25519_FILE_SIZE))
+        else:
+            with open(key_directory, 'rb') as f:
+                ED25519_KEY_POSITION = 32
+                f.seek(ED25519_KEY_POSITION)
+                rest_file = f.read()
+                EncodedValue = base64.b64encode(rest_file)
+                ed25519_id = EncodedValue.decode('utf-8').replace('=', '')
```

Comment:
Code style: CamelCase is for classes, but this is a variable, so it gets snake_case

```suggestion
                encoded_value = base64.b64encode(rest_file)
                ed25519_id = encoded_value.decode('utf-8').replace('=', '')
```